pkg/compact/compact.go

```diff
@@ -903,6 +922,10 @@ func (c *BucketCompactor) Compact(ctx context.Context) error {
 				continue
 			}
 
+			if err == emptyCompactedBlockSentinelError {
```

Comment:
So instead of this.. which is not an error, can we create `ULID` sentinel for moving forward instead?

Reply:
Yeah that's a bit nicer too, we don't have to introduce any special handling code in the caller now.  Just returning a fixed value instead now.